.github/workflows/manual-publish.yml

```diff
@@ -45,3 +50,32 @@ jobs:
           dry_run: ${{ inputs.dry_run }}
           aws_role: ${{ vars.AWS_ROLE_ARN }}
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  release-sdk-server-provenance:
+    needs: ['build']
+    permissions:
+      actions: read
+      id-token: write
+      contents: write
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.10.0
+    if: ${{ inputs.pkg_name == 'LaunchDarkly.ServerSdk' }}
+    with:
+      base64-subjects: "${{ needs.build.outputs.server-sdk-hashes }}"
+      upload-assets: true
+      upload-tag-name: ${{ input.tag_name }}
+      provenance-name: ${{ format('LaunchDarkly.ServerSdk-{0}_provenance.intoto.jsonl', input.tag_name) }}
```

Comment:
How do we specify release tag name for manual publish workflows?

Reply:
It is derived from the current version.  For example, the dotnet-server-sdk version is [here](https://github.com/launchdarkly/dotnet-server-sdk/blob/main/pkgs/sdk/server/src/LaunchDarkly.ServerSdk.csproj#L4).

Reply:
In C++ we made it so you could only publish an existing label. Which means you need to change the label if you are doing a manual publish to fix a problem. 

It is really inconvenient. Not sure if there would be a happy medium to handle both cases.